composer.json

```diff
@@ -59,7 +59,7 @@
         "symfony/phpunit-bridge": "^2.7 || ^3.0"
     },
     "suggest": {
-        "liip/imagine-bundle": "^0.9",
+        "liip/imagine-bundle": "If you want on-the-fly generate thumbnail or filter image(scale, crop, watermark..).",
```

Comment:
If you want on-the-fly thumbnail generation or image filtering (scale, crop, watermark…)